supervised/model_framework.py

```diff
@@ -202,6 +203,9 @@ def get_metric_name(self):
         self.metric_name = early_stopping.metric.name
         return early_stopping.metric.name
 
+    def get_metric(self):
+        return Metric({'name': self.get_metric_name()})
```

Comment:
Similar like in the ensemble.

Reply:
I don't think the `ModelFramework` has a `self.metric` attribute, if early stopping is used it is available via `early_stopping.metric`.  So I could do this:
```python 
def get_metric(self):
    early_stopping = self.callbacks.get("early_stopping")
    if early_stopping:
        return early_stopping.metric
    return Metric({'name': self.get_metric_name()})
```

Am I missing something?

Reply:
Makes sense!